docs/discover/view-document.asciidoc

```diff
@@ -1,25 +1,45 @@
-[[discover-document-context]]
-== View surrounding documents
+[[discover-view-document]]
+== View a document
 
-Once you've narrowed your search to a specific event in *Discover*,
-you can inspect the documents that occurred
-immediately before and after the event.
-To view the surrounding documents, your index pattern must contain time-based events.
+Once you've found a document of interest in Discover there are 2 more ways to view it.
+
+[float]
+[[discover-view-single-document]]
+=== View single document
 
 . In the document table, click the expand icon (>).
-. Click *View surrounding documents.*
+. Click **View single document**
 +
-In the context view, documents are sorted by the time field specified in the index pattern
+This allows you to view the document in two ways. **Table** displays the document's fields row by row.
+**JSON** (JavaScript Object Notation) allows you to take a look at how the document is returned from Elasticsearch.
++
+[role="screenshot"]
+image::images/discover-view-single-document.png[Discover single document view]
+
+The link is valid for the time the document is available in Elasticsearch. If you need another way to view single
+documents, you can extend **Table** and **JSON** by creating <<external-plugin-development, your own plugin>>.
+
+[float]
+[[discover-view-surrounding-documents]]
+=== View surrounding documents
+
+To view the surrounding documents, your index pattern must contain time-based events.
+
+. In the document table, click the expand icon (>).
+. Click **View surrounding documents**
+
+Now you can inspect the documents that occurred immediately before and after the event.
+
+Documents are sorted by the time field specified in the index pattern
 and displayed using the same set of columns as the *Discover* view from which
 the context was opened. The anchor document is highlighted in blue.
-+
+
 [role="screenshot"]
 image::images/discover-context.png[Image showing context view feature, with anchor documents highlighted in blue]
-+
+
 The filters you applied in *Discover* are carried over to the context view. Pinned 
 filters remain active, while normal filters are copied in a disabled state.
 
-+
```

Comment:
Also, add a "+" in the line above "To find the documents" so that the numbering of the steps is correct.